cc/position.cc

```diff
@@ -99,7 +99,8 @@ Position::UndoState Position::PlayMove(Coord c, Color color,
   return undo;
 }
 
-void Position::UndoMove(const UndoState& undo) {
+void Position::UndoMove(const UndoState& undo,
+                        ZobristHistory* zobrist_history) {
```

Comment:
doesn't UpdateLegalMoves need to use this?  or is this from a diff PR?

Reply:
I'm not sure what you mean. The code passes `zobrist_history` to `UpdateLegalMoves` at the end of this function